.env

```diff
@@ -121,7 +121,7 @@ SQL_SCRIPTS_PATH=
 # Data volumes
 #
 OPENMRS_CONFIG_CHECKSUMS_PATH=
-OPENMRS_DATA_PATH=
+OPENMRS_APPDATA_DIR=
 MYSQL_DATADIR=
 POSTGRES_DATADIR=
```

Comment:
How much of a big deal would be to renamed those two to `MYSQL_DATA_DIR` and `POSTGRES_DATA_DIR`? As a way of adopting consistent naming.

If complicated, forget about it.

Reply:
Done